Warning CA1416 when buildig for Android

<!--
# Please make sure that the issue is present in the
# develop branch of MonoGame before reporting
#
# You can download the development build installer from:
# http://www.monogame.net/downloads/
#
# GitHub issues are only for bug reports / feature requests
# if you have a question, ask it on the community site:
# https://community.monogame.net
-->

<!-- Write your issue below -->
Hi, there is alot of CA1416 warnings in build log when I try to build my project for Android.

`warning CA1416: This call site is reachable on: 'Android' 23.0 and later. 'XXX' is only supported on: 'Android' 31.0 and later.`
where 'XXX' is some entity from MonoGame like 'GamePadButtons.Back' or 'Game.Update(GameTime)'.

I think this happens because there is no `<SupportedOSPlatformVersion>23</SupportedOSPlatformVersion>` property in MonoGame.Framework.Android.csproj.

PS: Thank you fore releasing .Net 6 version, it is really cool.

<!-- System stats -->

#### What version of MonoGame does the bug occur on:
- MonoGame 3.8.1.303

#### What operating system are you using:
- macOS 12.4 ARM

#### What MonoGame platform are you using:
<!-- e.g. DesktopGL, WindowsDX, WindowsUWP, Android -->
- Android

In my case the warnings I get are related to API 21.

`warning CA1416: This call site is reachable on: 'Android' 21.0 and later. 'Vector2.Y' is only supported on: 'Android' 31.0 and later.`

--- a/MonoGame.Framework/MonoGame.Framework.Android.csproj
+++ b/MonoGame.Framework/MonoGame.Framework.Android.csproj
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0-android</TargetFramework>
+    <SupportedOSPlatformVersion>23</SupportedOSPlatformVersion>
     <DefineConstants>ANDROID;GLES;STBSHARP_INTERNAL</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <MGOpenGL>GLES</MGOpenGL>

